Explain importing Java functions from C. (#56)
graalvm/graal-languages-demos: graalwasm/graalwasm-embed-c-code-guide/README.md

}
```

## 4. Using Java functions from WebAssembly

You can also call Java functions from WebAssembly by importing them in your WebAssembly modules.
For the sake of an example, let's try to move the logic which computes the next element of the output (by doing an integer increment) to a Java function.

> Here is what we will need to add to our C file to declare an external function whose implementation we will provide in Java:

Reviewer: ```suggestion
We need to add the following to our C file to declare an external function whose implementation we will provide in Java:
```
